compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeNarrower.java

```diff
@@ -350,4 +366,49 @@ private SymbolEnv getTargetEnv(BLangNode targetNode, SymbolEnv env) {
 
         return targetEnv;
     }
+
+    private BFiniteType createFiniteType(BLangExpression expr) {
+        BTypeSymbol finiteTypeSymbol = Symbols.createTypeSymbol(SymTag.FINITE_TYPE,
+                Flags.asMask(EnumSet.noneOf(Flag.class)), Names.EMPTY, env.enclPkg.symbol.pkgID, null,
+                env.scope.owner, expr.pos, SOURCE);
+
+        BFiniteType finiteType = new BFiniteType(finiteTypeSymbol);
+        expr.type = symTable.getTypeFromTag(expr.type.tag);
+        finiteType.addValue(expr);
+        finiteTypeSymbol.type = finiteType;
+
+        return finiteType;
+    }
+
+    private void narrowTypeForEqualOrNotEqual(BLangBinaryExpr binaryExpr, BLangExpression lhsExpr, BLangExpression rhsExpr) {
```

Comment:
This line is longer than 120 characters.

Reply:
updated thanx